tests/AnalyzeSpec.hs

```diff
@@ -781,16 +781,14 @@ spec = describe "analyze" $ do
   describe "create-user-guard" $ do
     let code =
           [text|
-            (defun fail:bool (o:object{account})
-              (enforce false ""))
+            (defun fail:bool (x:integer)
```

Comment:
can we rename this to something not `fail`? even `predicate` would be fine. i'd also love to see a test of a user guard failing.

Reply:
Mistakenly kept the old name -- sorry about that